plugins/release/skills/release-migrate-variant-periodics/README.md

```diff
@@ -0,0 +1,350 @@
+# Release Migrate Periodics Skill
+
+This skill automates the migration of OpenShift periodic CI job definitions from one release version to another by transforming version-specific references and regenerating randomized cron schedules.
+
+## Overview
+
+Periodic jobs are CI tests that run on a schedule (via cron) rather than on pull requests. They are defined in YAML configuration files following the pattern `*-release-{major.minor}__periodics.yaml` in the `ci-operator/config/` directory of the openshift/release repository.
+
+When a new OpenShift release is branched, these periodic configurations need to be migrated to the new release version. This skill automates that process by:
+- Finding all periodic files for the source release
+- Transforming version-specific references
+- Regenerating randomized cron schedules
+- Creating new configuration files for the target release
+
+## Components
+
+### SKILL.md
+Claude Code skill definition that provides detailed implementation instructions for the AI assistant, including:
+- Input validation and normalization
+- File discovery patterns
+- Version string transformation rules
+- Cron schedule regeneration algorithm
+- Error handling strategies
+
+## Migration Transformations
+
+The skill performs the following transformations when migrating from version X.Y to version A.B:
+
+### 1. Base Images
+```yaml
+# Before (4.17)
+ocp_4_17_base-rhel9:
+  name: "4.17"
+  tag: base-rhel9
+
+# After (4.18)
+ocp_4_18_base-rhel9:
+  name: "4.18"
+  tag: base-rhel9
+```
+
+### 2. Builder Images
+```yaml
+# Before (4.17)
+ocp_builder_rhel-8-golang-1.22-openshift-4.17:
+  name: builder
+  tag: rhel-8-golang-1.22-openshift-4.17
+
+# After (4.18)
+ocp_builder_rhel-8-golang-1.22-openshift-4.18:
+  name: builder
+  tag: rhel-8-golang-1.22-openshift-4.18
+```
+
+### 3. Registry Paths
+```yaml
+# Before (4.17)
+- registry.ci.openshift.org/ocp/4.17:base-rhel9
+
+# After (4.18)
+- registry.ci.openshift.org/ocp/4.18:base-rhel9
+```
+
+### 4. Release Names
+```yaml
+# Before (4.17)
+releases:
+  latest:
+    integration:
+      name: "4.17"
+
+# After (4.18)
+releases:
+  latest:
+    integration:
+      name: "4.18"
+```
+
+### 5. Branch Metadata
+```yaml
+# Before (4.17)
+zz_generated_metadata:
+  branch: release-4.17
+  org: openshift
+  repo: cloud-credential-operator
+
+# After (4.18)
+zz_generated_metadata:
+  branch: release-4.18
+  org: openshift
+  repo: cloud-credential-operator
+```
+
+### 6. Cron Schedules
+```yaml
+# Before (randomized for 4.17)
+cron: 11 4 14 8 *
+
+# After (re-randomized for 4.18)
+cron: 37 15 14 8 *
```

Comment:
Do we want to tell the model that some periodics have `interval` and not `cron`?